### DISCUSSION TITLE
Amplify + web3auth

### DISCUSSION THREAD

**Discussion by Shusanta** [Web3Auth/Web3Auth#691] [Help]
I saw there was a cognito guide, would love some docs on having it work with app sync or even an all around aws amplify guide

**Comment by shahbaz17** [comment]
Hey @Shusanta 

Thanks for making this request. Currently, there's no such plan. But I would love to make this guide in future.

How about you taking the lead in creating an authentication for a web application using Amplify, and I'll help you add Web3Auth to it?
